client/coral-framework/reducers/notification.js

```diff
@@ -1,24 +0,0 @@
-import * as actions from '../actions/notification';
-import {Map} from 'immutable';
-
-const initialState = Map({
-  text: '',
-  type: '',
-  position: 400
-});
-
-export default (state = initialState, action) => {
-  switch (action.type) {
```

Comment:
it's not being used anywhere. I deleted the `<Notification>` component, so nothing was reading its state.